src/ConfigContext.ts

```diff
@@ -93,6 +94,10 @@ export async function initializeConfiguration(): Promise<ConfigContext> {
     }
     // Allow override of platform value with URL query parameter
     const params = new URLSearchParams(window.location.search);
+    if (params.has("armAPIVersion")) {
```

Comment:
Consider making it all lowercase since query params are case-insensitive (right?)

Reply:
They are case sensitive. Headers are not